server.py

```diff
@@ -192,25 +210,27 @@ def createSession(username):
 def login():
     username = bottle.request.forms.get("username")
     password = bottle.request.forms.get("password")
-    with shelve.open(user_db) as users:
-        if not username in users:
-            return bottle.template("home.html", message="User does not exist.")
-        if users[username].password != password:
-            return bottle.template("home.html", message="Incorrect password.")
+    if (
```

Comment:
this would transfer the record and then compute the length. Let's use [peewee's count ](http://docs.peewee-orm.com/en/latest/peewee/querying.html#counting-records)

Reply:
in fact a better way would be [the exists function](http://docs.peewee-orm.com/en/latest/peewee/api.html?highlight=exists#SelectBase.exists)